demo-project/src/demo_project/pipelines/reporting/nodes.py

```diff
@@ -56,7 +58,12 @@ def make_price_histogram(model_input_data: pd.DataFrame) -> go.Figure:
         BaseFigure: Plotly object which is serialised as JSON for rendering
     """
     price_data_df = model_input_data[["price", "engine_type"]]
-
+    p1 = np.random.randint(100, 300) / 1000
+    p2 = np.random.randint(100, 400) / 1000
```

Comment:
```suggestion
    p1 = np.random.uniform(0.1, 0.3)
    p2 = np.random.uniform(0.1, 0.4)
```

Or you could just generate a 3-dimensional probability distribution like this:
```
p = np.random.dirichlet([1, 1, 1])
```
(FYI this works because a Dirichlet can be used to generate set of probability distributions. Try it out to see what that does. `sum(p)` will always be 1 with each value value positive. This is basically what defines a probability distribution)